gperftools-toolfile.spec

```diff
@@ -29,4 +29,15 @@ cat << \EOF_TOOLFILE >%{i}/etc/scram.d/tcmalloc.xml
 </tool>
 EOF_TOOLFILE
 
+cat << \EOF_TOOLFILE >%{i}/etc/scram.d/gperf.xml
+<tool name="gperf" version="@TOOL_VERSION@">
+  <lib name="profiler"/>
+  <client>
+    <environment name="GPERF_BASE" default="@TOOL_ROOT"/>
```

Comment:
@gartung , you are missing a `@` at th end of  `@TOOL_ROOT` . Please update and try start the tests